src/Math.php

```diff
@@ -52,6 +52,9 @@ public static function ceilFloat($n) : int{
 	 * @return float[]
 	 */
 	public static function solveQuadratic(float $a, float $b, float $c) : array{
+		if((int) $a === 0){
```

Comment:
this causes unexpected behaviour if `a` is between 0 and 1.

Reply:
Fixed with `$a === (float) 0`, because `$a === 0` returns `false`.